Docs: Update the coverage of fine-grained access control (#41398)
grafana/grafana: docs/sources/enterprise/access-control/_index.md

## Resources with fine-grained permissions

> Fine-grained access control is currently available for the following capabilities:

Reviewer: ```suggestion
Fine-grained access control is available for the following capabilities:
```
"Currently" is covered by the version number, so that term is redundant in nature.

> - [Explore]({{< relref "../../explore/_index.md" >}}) 

Reviewer 2: ```suggestion
- [Use Explore mode]({{< relref "../../explore/_index.md" >}}) 
```

> - [Managing users]({{< relref "../../manage-users/_index.md" >}})

Reviewer 2: ```suggestion
- [Manage users]({{< relref "../../manage-users/_index.md" >}})
```

> - [Managing Users]({{< relref "../../manage-users/_index.md" >}})

Reviewer: ```suggestion
- [Managing users]({{< relref "../../manage-users/_index.md" >}})
```

Reviewer: Change title of file if `U` if uppercase.

- [Manage LDAP authentication]({{< relref "../../auth/ldap/_index.md" >}})
- [Manage data sources]({{< relref "../../datasources/_index.md" >}})
- [Manage data source permissions]({{< relref "../datasource_permissions.md" >}})

> - [Manage Grafana Enterprise license]({{< relref "../license/_index.md" >}})

Reviewer: ```suggestion
- [Manage a Grafana Enterprise license]({{< relref "../license/_index.md" >}})
```

Reviewer: Change title of file as well.

Author: Not sure I understand, what file?

> - [Provisioning Grafana]({{< relref "../../administration/provisioning/_index.md" >}})

Reviewer 2: ```suggestion
- [Provision Grafana]({{< relref "../../administration/provisioning/_index.md" >}})
```

> - [Reporting]({{< relref "../reporting.md" >}})

Reviewer 2: ```suggestion
- [Manage reports]({{< relref "../reporting.md" >}})
```

- [View server information]({{< relref "../../administration/view-server/_index.md" >}})

> To learn more about specific endpoints where you can use access control, refer to [Permissions]({{< relref "./permissions.md" >}}) and to the relevant [API]({{< relref "../../http_api/_index.md" >}}) guide:

Reviewer: ```suggestion
To learn about specific endpoints where you can use fine-grained access control, refer to [Permissions]({{< relref "./permissions.md" >}}) and to the relevant [API]({{< relref "../../http_api/_index.md" >}}) documentation:
```

> To learn about specific endpoints where you can use fine-grained access control, refer to [Permissions]({{< relref "./permissions.md" >}}) and to the relevant [API]({{< relref "../../http_api/_index.md" >}}) documentation:

Reviewer: ```suggestion
To learn about specific endpoints where you can use fine-grained access control, refer to [Permissions]({{< relref "./permissions.md" >}}) and to the relevant [API]({{< relref "../../http_api/_index.md" >}}) documentation.
```
